.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: 'User Reported Bug: {Insert reported bug here}'
+labels: bug
+assignees: ''
```

Comment:
Tried to do it, but not sure if it will work

![image](https://github.com/ai-cfia/nachet-frontend/assets/96267006/fb12923f-9eb3-4418-a25d-ad4eec8ac762)

Reply:
Remove nachet from assignee and keep it empty since we can't assign Nachet team on it.